lua/wire/client/text_editor/texteditor.lua

```diff
@@ -2631,13 +2673,50 @@ function EDITOR:AC_FillList()
 	local maxw = 15
 
 	surface.SetFont( "E2SmallFont" )
+	
+	-- PickColorFind
+	local PC_TXT = "Open PickColor"
+	if PickColorShouldDraw == true then
+		local pc_text = vgui.Create("DLabel")
+		pc_text:SetText( "" )
+		
+		-- Override paint to give it the "E2 theme" and to make it highlight when selected
```

Comment:
This is copy-pasted from below. If you have a lot of GUI elements that should be the same or similar, then use `vgui.CreateFromTable` (as code elsewhere in the text editor uses).